python/chemiscope/explore.py

```diff
@@ -197,6 +198,121 @@ def soap_pca_featurize(frames, environments=None):
     return pca.fit_transform(feats)
 
 
+def metatensor_featurizer(model, extensions_directory="extensions/", device=None):
```

Comment:
Default should be None, like for all other integrations


```suggestion
def metatensor_featurizer(model, extensions_directory=None, device=None):
```